src/main/java/com/notably/logic/parser/AddCommandParser.java

```diff
@@ -30,7 +29,7 @@ public AddCommand parse(String args) throws ParseException {
 
         if (!arePrefixesPresent(argMultimap, PREFIX_NAME, PREFIX_ADDRESS, PREFIX_PHONE, PREFIX_EMAIL)
                 || !argMultimap.getPreamble().isEmpty()) {
-            throw new ParseException(String.format(MESSAGE_INVALID_COMMAND_FORMAT, AddCommand.MESSAGE_USAGE));
+            throw new ParseException(String.format("MESSAGE_INVALID_COMMAND_FORMAT", AddCommand.MESSAGE_USAGE));
```

Comment:
Hey johanna! That's not how `String.format` works :( Take a look at the documentation of it [here](https://docs.oracle.com/en/java/javase/11/docs/api/java.base/java/lang/String.html#format(java.lang.String,java.lang.Object...))

In short, this should be written as `throw new ParseException("MESSAGE_INVALID_COMMAND_FORMAT")` instead